src/main/java/mat/client/shared/MeasureLibraryGridToolbar.java

```diff
@@ -0,0 +1,163 @@
+package mat.client.shared;
+
+import java.util.Collection;
+
+import org.gwtbootstrap3.client.ui.Button;
+import org.gwtbootstrap3.client.ui.constants.ButtonSize;
+import org.gwtbootstrap3.client.ui.constants.ButtonType;
+import org.gwtbootstrap3.client.ui.constants.IconType;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.thirdparty.guava.common.annotations.VisibleForTesting;
+import mat.client.measure.ManageMeasureSearchModel;
+
+public class MeasureLibraryGridToolbar extends HorizontalFlowPanel {
+
+    private Button versionButton;
+    private Button historyButton;
+    private Button editButton;
+    private Button shareButton;
+    private Button cloneButton;
+    private Button exportButton;
+
+    public MeasureLibraryGridToolbar() {
+        setStyleName("action-button-bar");
+        addStyleName("btn-group");
+        addStyleName("btn-group-sm");
+
+        versionButton = GWT.create(Button.class);
+        historyButton = GWT.create(Button.class);
+        editButton = GWT.create(Button.class);
+        shareButton = GWT.create(Button.class);
+        cloneButton = GWT.create(Button.class);
+        exportButton = GWT.create(Button.class);
+
+        add(versionButton);
+        add(historyButton);
+        add(editButton);
+        add(shareButton);
+        add(cloneButton);
+        add(exportButton);
+
+
+        applyDefault();
+    }
+
+    @VisibleForTesting
+    void applyDefault() {
+        applyDefaultAllButExport();
+
+        exportButton.setText("Export");
+        exportButton.setTitle("Click to export");
+        exportButton.setWidth("72px");
+        buildActionButton(exportButton, IconType.DOWNLOAD);
+    }
+
+    private void applyDefaultAllButExport() {
+        versionButton.setText("Create Version/Draft");
+        versionButton.setTitle("Click to create version or draft");
+        versionButton.setWidth("146px");
```

Comment:
Maybe all these 3 line can go into buildActionButton(...)

Reply:
Yeah, I hesitated because of the number of arguments. 5 args is quite high. But there are no other good options. I will move it to the method.